crosswalks/Citation_File_Format_1.2.0.README.md

```diff
@@ -0,0 +1,243 @@
+
+- `url`, when the URL points to a URL where the software is installed
+- `repository`, when the repository provides an installed artifact of the software
+- `identifiers[i].type==url`, when the resolved URL is to an installed artifact of the software
+
+### `softwareRequirements`/`softwareSuggestions` <=> `references`
+
+- CodeMeta `softwareRequirements`: "Required software dependencies"
+- CodeMeta `softwareSuggestions`: "Optional dependencies , e.g. for optional features, code development, etc"
+- CFF `references`: "Reference(s) to other creative works."
+
+#### CodeMeta to CFF
+
+The scope of `references` could be documented more clearly in CFF (<https://github.com/citation-file-format/citation-file-format/issues/327>).
+CFF's `references` is meant to work like a references list for a paper, 
+e.g., should include prior work that the work builds on.
+Required dependencies are interpreted to clearly constitute prior work that software builds on.
+Therefore, everything that is in `softwareRequirements` should go into `references`, but not the other way round.
+
+Optional dependencies and dependencies needed for code development could be assumed 
+to build on the software (i.e., require prior existence of the software).
+Therefore, the software would constitute a `reference` *from* the `softwareSuggestion`,
+but not the other way round.
+I.e., `softwareSuggestions` should *not* be included in `references`.
+
+#### CFF to CodeMeta
+
+Not all CFF `references` can be assumed to be `softwareRequirements`,
+as `references` can contain arbitrary other work that the software builds on.
+The same is true for `softwareSuggestions`.
+`references` therefore should not be included automatically in either, 
+without further manual verification.
+
+### `author` <=> `authors`
+
+CFF `authors` is an array of CFF `person` or CFF `entity` objects.
+
+### `citation` <=> `preferred-citation`/`references`/CFF root document
+
+- CodeMeta `citation`: "A citation or reference to another creative work, such as another publication, web page, scholarly article, etc."
+- CFF `preferred-citation`: "A reference to another work that should be cited instead of the software or dataset itself."
+- CFF `references`: "Reference(s) to other creative works."
+
+#### CodeMeta to CFF
+
+- When the `citation` is a citation to another work describing the software (a "meta" work, e.g., a software paper in JOSS, a traditional paper about the software), it maps to CFF `preferred-citation`.
+- When the `citation` is to a work not describing the software itself, it maps to an entry in CFF `references`
+
+#### CFF to CodeMeta
+
+CFF `preferred-citation` maps to CodeMeta `citation`.
+Any given entry in CFF `references` may map to CodeMeta `citation`, given the latter's definition as "A citation or reference to another creative work, such as another publication, web page, scholarly article, etc."
+
+### `contributors` <=> potentially `authors`
```

Comment:
I think this shouldn't be mapped to authors, since there is already a author property.
The reason for this, is that authors in CFF files are listed for credit purposes and contributors might not be eligible for this credit.

Reply:
In schema.org contributors are regarded as a secondary contributor:
https://schema.org/contributor

Reply:
Okay, fair enough. I'll take this out again.

Reply:
Fixed in 286fefa7eea6c089176d211bf0f641469b1e685b.